test/test_shared.cpp

```diff
@@ -2082,7 +2082,7 @@ TEST_IF(Shared_AsyncMultiprocess, allow_async)
 
 #if !defined(_WIN32)
 // this test does not work with valgrind:
```

Comment:
AFAIK Jenkins is running valgrind on linux so if it works for you maybe it's ok now, we'll find out if you merge 😄